app/src/pages/inside/testCaseLibraryPage/createTestCaseModal/testCaseDetails/steps/steps.scss

```diff
@@ -64,3 +104,46 @@
     height: 24px;
   }
 }
+
+@keyframes dropHighlight {
+  0% {
+    box-shadow: 0 0 0 3px rgba($COLOR--topaz-focused, 0.4);
+    transform: scale(1.01);
+  }
+
+  50% {
+    box-shadow: 0 0 0 6px rgba($COLOR--topaz-focused, 0.2);
+  }
+
+  100% {
+    box-shadow: 0 0 0 0 rgba($COLOR--topaz-focused, 0);
+    transform: scale(1);
+  }
+}
+
+.drag-preview {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 8px;
+  width: 376px;
+  height: 52px;
+  padding-left: 12px;
+  background-color: var(--rp-ui-base-bg-000);
+  border: 1px solid $COLOR--e-200;
```

Comment:
I believe we should use colors from the latest color palette, specifically those defined as --rp-ui-... variables.

Reply:
Thanks for information. Fixed.